__tests__/app.test.js

```diff
@@ -44,13 +44,142 @@ describe('GET /api/topics', () => {
       });
   });
 });
-describe('GET /api/topisc', () => {
+describe('ERROR GET /api/topicd', () => {
   test('sends GET to incorrect endpoint -> returns 404', () => {
     return request(app)
       .get('/api/topisc')
       .expect(404)
   });
 });
 
+describe('GET /api/articles/:article_id', () => {
+  test('sends GET to ID endpoint -> returns an object', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+        const output = res.body.rows[0];
```

Comment:
Your result should already be a single object here, instead of an array that you're having to use [0] on. Please revisit your model.